### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by Evexal** [w-okada/voice-changer#792] [Q&A]
Question, when press gpu, i only see cpu, i use amd rx 6400
![image](https://github.com/w-okada/voice-changer/assets/144136628/9bb4b9de-6d9d-42ae-ac4a-ffb73f7f46b1)
![image](https://github.com/w-okada/voice-changer/assets/144136628/5e3f26ae-ba90-40d4-b360-6996ac7e5738)
There any way to fix it? Or need to wait for a fix?

**Comment by korovagg** [comment]
The same problem.
